src/commands/docs.js

```diff
@@ -25,27 +26,191 @@ function readXmirsRecursively(dir) {
+              <xsl:call-template name="break">
+                <xsl:with-param name="text" select="//comments/comment[@line = current()/@line]"/>
+              </xsl:call-template>
+            </p>
+          </div>
+        </xsl:if>
+      </xsl:for-each>
+    </div>
+  </xsl:template>
+
+  <xsl:template name="break">
+    <xsl:param name="text" select="string(.)"/>
+    <xsl:choose>
+      <xsl:when test="contains($text, '\\n')">
+        <xsl:value-of select="substring-before($text, '\\n')"/>
+        <br/>
+        <xsl:call-template name="break">
+          <xsl:with-param 
+            name="text" 
+            select="substring-after($text, '\\n')"
+          />
+        </xsl:call-template>
+      </xsl:when>
+      <xsl:otherwise>
+        <xsl:value-of select="$text"/>
+      </xsl:otherwise>
+    </xsl:choose>
+  </xsl:template>
+</xsl:stylesheet>`;
+
+/**
+ * Creates documentation block from given XMIR
+ * @param {String} path - path of XMIR
+ * @return {String} HTML block
+ */
+function createXmirHtmlBlock(path) {
+  try {
+    const xmir = fs.readFileSync(path).toString();
+    return transformDocument(xmir, BLOCK_XSL);
+  } catch(error) {
+    throw new Error('Error while applying XSL to XMIR:', error);
+  }
+}
+
+/**
+ * Generates Package HTML
+ * @param {String} name - Package name
+ * @param {String[]} xmir_htmls - Array of xmirs htmls
+ * @param {String} css_path - CSS file path
+ * @return {String} HTML of the package
+ */
+function generatePackageHtml(name, xmir_htmls, css_path) {
+  const title = `<h1 class="package-title">Package ${name} documentation</h1>`;
+  return `<!DOCTYPE html>
+    <html>
+      <head>
+        <link href="${css_path}" rel="stylesheet" type="text/css">
+        ${title}
+      </head>
+      <body>
+        ${xmir_htmls.join('\n')}
+      </body>
+    </html>`;
+}
+
+/**
+ * Wraps given html body
+ * @param {String} html - HTML body
+ * @param {String} css_path - CSS file path
+ * @return {String} Ready HTML
+ */
+function wrapHtml(html, css_path) {
+  return `
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <link href="${css_path}" rel="stylesheet" type="text/css">
+      </head>
+      <body>
+        ${html}
+      </body>
+    </html>
+  `;
+}
+
 /**
  * Command to generate documentation.
  * @param {Hash} opts - All options
  */
-module.exports = function(opts) {
+module.exports = async function(opts) {
   try {
-    const input = path.resolve(opts.target, '.eoc', '1-parse');
+    const input = path.resolve(opts.target, '1-parse');
     const output = path.resolve(opts.target, 'docs');
     fs.mkdirSync(output, {recursive: true});
+
+    const css = path.join(output, 'styles.css');
+    fs.writeFileSync(css, '');
+
+    const packages_info = {};
+    const all_xmir_htmls = [];
+
     const xmirs = readXmirsRecursively(input);
     for (const xmir of xmirs) {
       const relative = path.relative(input, xmir);
+      const name = path.parse(xmir).name;
+
```

Comment:
@danben2904 we're not using empty lines inside method body: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html